rend3-egui/src/lib.rs

```diff
@@ -87,6 +88,72 @@ impl EguiRenderRoutine {
                 .unwrap();
         });
     }
+
+    /// Creates a wgpu texture
+    pub fn create_wgpu_texture(
```

Comment:
At this point, this is basically just a call to create_texture -- what I meant more was this function would create a texture, then internally call wgpu_texture_to_egui. This would allow the user to not care about texture allocation